lib/insights_cloud/async/insights_full_sync.rb

```diff
@@ -28,7 +29,7 @@ def plan(organizations)
         end
       end
 
-      def run
+      def action
```

Comment:
While I'm going on about naming, this feels a bit unfortunate too. I mean a method called `action` on an action class does not really convey the intent